css/style.css

```diff
@@ -9,285 +14,633 @@ body {
 .new-сollection__menu {
-    position: relative;
+  position: relative;
 }
 
 .logo {
-    margin-right: 238px;
-    max-width: 98px;
-    max-height: 37px;
-    margin-top: 40px;
-    margin-left: 52px;
+  margin-right: 238px;
+  max-width: 98px;
+  max-height: 37px;
+  margin-top: 40px;
+  margin-left: 52px;
 }
 
 .logo-mobil {
-    position: absolute;
-    width: 98px;
-    height: 37px;
-    padding: 5% 0 0 20%; 
+  position: absolute;
+  width: 98px;
+  height: 37px;
+  padding: 5% 0 0 20%; 
 }
 
 .lang {
-    margin-top: 42px;
+  margin-top: 42px;
 }
 
 .new-сollection__menu {
-    width: 92px;
-    height: 92px;
-    position: absolute;
-    top: 0;
-    right: 0;
+  width: 92px;
+  height: 92px;
+  position: absolute;
+  top: 0;
+  right: 0;
  }
 
  .new-сollection__menu-box {
-    opacity: 0.05;
-    background: #000;
-    width: 92px;
-    height: 92px;
+  opacity: 0.05;
+  background: #000;
+  width: 92px;
+  height: 92px;
  }
 
  .menu {
-     position: absolute;
-     top: 40%;
-     right: 30%; 
+  position: absolute;
+  top: 40%;
+  right: 30%; 
  }
 
  .girl {
-    max-width: 855.54px;
-    height: 839px;
-    left: 49.28px;
-    top: 60.92px;
-    margin-left: 50px;
-    background: url(../img/girl.png) no-repeat center top 100px;
+  max-width: 855.54px;
+  height: 839px;
+  left: 49.28px;
+  top: 60.92px;
+  margin-left: 50px;
+  background: url(../img/girl.png) no-repeat center top 100px;
  }
 
 .title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 80px;
-    line-height: 82px;
-    position: absolute;
-    width: 475px;
-    height: 164px;
-    left: 789px;
-    top: 210px;
-    margin: 0;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 80px;
+  line-height: 82px;
+  position: absolute;
+  width: 475px;
+  height: 164px;
+  left: 789px;
+  top: 210px;
+  margin: 0;
 }
 
 .text {
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    color: #000000;
-    position: absolute;
-    width: 467px;
-    height: 127px;
-    left: 789px;
-    top: 372px;
-    margin-bottom: 36px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  color: #000000;
+  position: absolute;
+  width: 467px;
+  height: 127px;
+  left: 789px;
+  top: 372px;
+  margin-bottom: 36px;
 }
 
 .new-сollection__main-card {
-    position: absolute;
-    left: 789px;
-    top: 550px;
-    display: flex;
-    width: 475px;
-    align-items: center;
-    justify-content: space-between;
+  position: absolute;
+  left: 789px;
+  top: 550px;
+  display: flex;
+  width: 475px;
+  align-items: center;
+  justify-content: space-between;
 }
 
 .button {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 13px;
-    line-height: 0;
-    border: 1.5px solid #000;
-    background-color: #fff;
-   
-     
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 0;
+  border: 1.5px solid #000;
+  background-color: #fff;  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5449357)